src/main/java/uk/gov/justice/digital/hmpps/keyworker/dto/PrisonConfigRequest.kt

```diff
@@ -16,6 +15,16 @@ data class PrisonConfigResponse(
   val hasPrisonersWithHighComplexityNeeds: Boolean,
   val allowAutoAllocation: Boolean,
   val capacity: Int,
-  val maximumCapacity: Int,
   val frequencyInWeeks: Int,
-)
+) {
+  companion object {
+    val DEFAULT =
+      PrisonConfigResponse(
+        isEnabled = false,
+        hasPrisonersWithHighComplexityNeeds = false,
+        allowAutoAllocation = false,
+        capacity = 6,
```

Comment:
Since this replaces the previous max capacity, should this default be 9 rather than 6?